src/test/ui/span/impl-wrong-item-for-trait.stderr

```diff
@@ -0,0 +1,56 @@
+error[E0323]: item `bar` is an associated const, which doesn't match its trait `<FooConstForMethod as Foo>`
```

Comment:
Not related to this PR, I think, but this text "`<FooConstForMethod as Foo>`" seems pretty funny to me; I expect to see just `Foo`... is there an open issue on that?